### PR TITLE
Update Windows documentation

### PR DESCRIPTION
**Pull request by davidemyers**
## Description

This PR revises the Windows installation instructions in order to hopefully avoid any further permissions problems.

## Motivation and Context

There have been many issues opened as a result of WSL permissions problems.

## How Has This Been Tested?

The revised instructions were tested by @TC1977.

## Types of changes

- Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/deploy-from-windows.md [hunk 1]
```diff
@@ -17,17 +24,25 @@ Wait a minute for Windows to install a few things in the background (it will eve
...
+
+You may need to follow [these directions](https://devblogs.microsoft.com/commandline/copy-and-paste-arrives-for-linuxwsl-consoles/) in order to paste commands into the Ubuntu Terminal.
 
 ```shell
-sudo apt-get update && sudo apt-get install git build-essential libssl-dev libffi-dev python3-dev python3-pip python3-setuptools python3-virtualenv -y
+cd
+umask 0002
+sudo apt update
+sudo apt install -y python3-virtualenv
+git clone https://github.com/trailofbits/algo
```
Comment (jackivanov) [inline docs/deploy-from-windows.md]: Is `git` pre installed by default on WSL?

File: docs/deploy-from-windows.md [hunk 2]
```diff
@@ -17,17 +24,25 @@ Wait a minute for Windows to install a few things in the background (it will eve
...
+cd algo
 ```
 
-Clone the Algo repository:
+Now you can continue by following the [README](https://github.com/trailofbits/algo#deploy-the-algo-server) from the 4th step to deploy your Algo server!
 
+You'll be instructed to edit the file `config.cfg` in order to specify the Algo user accounts to be created. If you're new to Linux the simplest editor to use is `nano`. To edit the file while in the `algo` directory, run:
 ```shell
-cd ~ && git clone https://github.com/trailofbits/algo && cd algo
+nano config.cfg
```
Comment (jackivanov) [inline docs/deploy-from-windows.md]: Can we use here `editor` instead of `nano`?

File: docs/deploy-from-windows.md (hunk 1 above)
Comment (davidemyers) [reply docs/deploy-from-windows.md]: I don't believe @TC1977 needed to install `git`.

File: docs/deploy-from-windows.md (hunk 2 above)
Comment (davidemyers) [reply docs/deploy-from-windows.md]: I think hiding the true name of the program from new users will hinder them if they need to Google for help. And anyone who knows enough to have used `update-alternatives` will likely disregard this suggestion anyway.

File: docs/deploy-from-windows.md (hunk 1 above)
Comment (TC1977) [reply docs/deploy-from-windows.md]: I didn't need to install `git`, and I believe [that's the default for WSL2 at least](https://docs.microsoft.com/en-us/windows/nodejs/setup-on-wsl2#set-up-git-optional).

File: docs/deploy-from-windows.md (hunk 2 above)
Comment (TC1977) [reply docs/deploy-from-windows.md]: `nano` worked fine for me.

File: docs/deploy-from-windows.md (hunk 1 above)
Comment (TC1977) [reply docs/deploy-from-windows.md]: Ok, looks like I can't install WSL2 currently, as I'm not an "Insider" and only have Win10 OS build 18362. Well I reinstalled WSL and Ubuntu 18.04.2 LTS, and `git` still comes pre-installed, as does `nano`.

**Pull request merged**